pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/SegmentPreprocessingJob.java

```diff
@@ -68,25 +67,20 @@
 
 import static org.apache.hadoop.mapreduce.MRJobConfig.*;
 import static org.apache.hadoop.security.UserGroupInformation.*;
-import static org.apache.pinot.hadoop.job.InternalConfigConstants.*;
 
 
 /**
  * A Hadoop job which provides partitioning, sorting, and resizing against the input files, which is raw data in Avro format.
  * Thus, the output files are partitioned, sorted, resized after this job.
  * In order to run this job, the following configs need to be specified in job properties:
- * * enable.partitioning: false by default. If enabled, this job will fetch the partition config from table config.
- * * enable.sorting: false by default. If enabled, the job will fetch sorted column from table config.
- * * enable.resizing: false by default. If partitioning is enabled, this is force to be disabled. If enabled, min.num.output.files is required.
- * * min.num.output.files: null by default. It's required if resizing is enabled.
- * * max.num.records: null by default. The output file will be split into multiple files if the number of records exceeded max.num.records.
+ * * enable.preprocessing: false by default. Enables preprocessing job.
+ * * preprocess.num.files: null by default. Number of files output.
```

Comment:
**preprocess.num.files** and **partition.max.records.per.file** are not from job properties but from table config